sdk/examples/intkey_python/sawtooth_intkey/client_cli/intkey_cli.py

```diff
@@ -0,0 +1,353 @@
+        clog.setLevel(logging.DEBUG)
+
+    return clog
+
+
+def setup_loggers(verbose_level):
+    logger = logging.getLogger()
+    logger.setLevel(logging.DEBUG)
+    logger.addHandler(create_console_handler(verbose_level))
+
+
+def create_parent_parser(prog_name):
+    parent_parser = argparse.ArgumentParser(prog=prog_name, add_help=False)
+    parent_parser.add_argument(
+        '-v', '--verbose',
+        action='count',
+        help='enable more verbose output')
+
+    try:
+        version = pkg_resources.get_distribution(DISTRIBUTION_NAME).version
+    except pkg_resources.DistributionNotFound:
+        version = 'UNKNOWN'
+
+    parent_parser.add_argument(
+        '-V', '--version',
+        action='version',
+        version=(DISTRIBUTION_NAME + ' (Hyperledger Sawtooth) version {}')
+        .format(version),
+        help='print version information')
+
+    return parent_parser
+
+
+def create_parser(prog_name):
+    parent_parser = create_parent_parser(prog_name)
+
+    parser = argparse.ArgumentParser(
+        parents=[parent_parser],
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    subparsers = parser.add_subparsers(title='subcommands', dest='command')
+
+    add_set_parser(subparsers, parent_parser)
+    add_inc_parser(subparsers, parent_parser)
+    add_dec_parser(subparsers, parent_parser)
+    add_show_parser(subparsers, parent_parser)
+    add_list_parser(subparsers, parent_parser)
+
+    add_generate_parser(subparsers, parent_parser)
+    add_load_parser(subparsers, parent_parser)
+    add_populate_parser(subparsers, parent_parser)
+    add_create_batch_parser(subparsers, parent_parser)
+    add_workload_parser(subparsers, parent_parser)
+
+    return parser
+
+
+def add_set_parser(subparsers, parent_parser):
+    parser = subparsers.add_parser('set', parents=[parent_parser])
+
+    parser.add_argument(
+        'name',
+        type=str,
+        help='the name of the key to set')
+
+    parser.add_argument(
+        'value',
+        type=int,
+        help='the value to set')
+
+    parser.add_argument(
+        '--url',
+        type=str,
+        help='the url of the REST API')
+
+    parser.add_argument(
+        '--keyfile',
+        type=str,
+        help="the file containing the user's private key")
+
+    parser.add_argument(
+        '--wait',
+        nargs='?',
+        const=sys.maxsize,
+        type=int,
+        help='seconds to wait for the transaction to commit')
+
+
+def do_set(args):
+    name, value, wait = args.name, args.value, args.wait
+    client = _get_client(args)
+    response = client.set(name, value, wait)
+    print(response)
+
+
+def add_inc_parser(subparsers, parent_parser):
+    parser = subparsers.add_parser('inc', parents=[parent_parser])
+
+    parser.add_argument(
+        'name',
+        type=str,
+        help='the name of the key to increment')
+
+    parser.add_argument(
+        'value',
+        type=int,
+        help='the value to increment')
```

Comment:
Minor comment: Because the argument name is "value", I'd prefer "the amount to increment" (or other word that isn't "value").  Tech-writer best practice is to avoid circular definitions by using a different word in the description. 

This comment also applies to "set value" above and "decrement value" below.